packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -270,6 +270,8 @@ export abstract class TestNode extends vscode.TreeItem {
     dark: DARK_BLUE_BUTTON
   };
 
+  // TODO: create a ticket to address this particular issue.
+  // @ts-ignore
```

Comment:
this was a weird case with extending a class from another module.  I didn't find a good work around so I went the heavy handed route.